src/lib.rs

```diff
@@ -2,7 +2,7 @@
 #![deny(missing_debug_implementations)]
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(all(test, feature = "nightly"), feature(test))]
-#![doc(html_root_url = "https://docs.rs/headers/0.4.0")]
+#![doc(html_root_url = "https://docs.rs/headers/0.4.1")]
```

Comment:
This can be removed entirely, crates.io has been linking to docs.rs by default for years.